bedrock/settings/base.py

```diff
@@ -1796,35 +1796,83 @@ def before_send(event, hint):
                 "id": "price_1LYCqOJNcmPzuWtRuIXpQRxi",
                 "price": 2.00,
                 "currency": "CHF",
+                "analytics": {
+                    "brand": "relay",
+                    "plan": "relay-email",
+                    "currency": "CHF",
+                    "discount": "0",
+                    "price": "2.00",
+                    "period": "monthly",
+                },
             },
             "yearly": {
                 "id": "price_1LYCqyJNcmPzuWtR3Um5qDPu",
                 "price": 1.00,
                 "currency": "CHF",
+                "analytics": {
+                    "brand": "relay",
+                    "plan": "relay-email",
+                    "currency": "CHF",
+                    "discount": "6.00",
+                    "price": "12.00",
```

Comment:
**question**: Curious why the price is `12.00` here but above in the same object the price is `1.00`?

Reply:
Yearly vs monthly pricing (if I understand the question properly). At checkout for monthly pricing they pay 2.00 at checkout for yearly they pay 12.00.